java/mykola-melnyk/src/com/zoolatech/lecture5/tasks/_3/Task3.java

```diff
@@ -0,0 +1,29 @@
+package com.zoolatech.lecture5.tasks._3;
+
+import java.util.*;
```

Comment:
please don't use `*` import

Reply:
This is intellij IDEA that merges multiple imports with *. What should I do with this?

Reply:
You can change this behaviour in Intellij Idea preferences: Editor -> Code Style -> Java. "Class count to use import with '*'" and "Names count to use static import with '*'" properties allow to specify the min amount of import statements when the '*' import should be used. If you change values of these properties to some big number (e.g., 99) '*' import won't be applied for almost all cases.

<img width="1002" alt="Screenshot 2022-08-24 at 22 05 41" src="https://user-images.githubusercontent.com/108929952/186503253-558f5e02-daf1-48ed-b408-dbc272b6ef64.png">

Reply:
Thank you, I've applied the recoommended settings.